scripts/perftests/RunPerformanceTestRunner.ps1

```diff
@@ -0,0 +1,47 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$resultsDirectoryPath,
+    [string]$nugetClientFilePath,
+    [string]$testDirectoryPath,
+    [string]$logsDirectoryPath,
+    [switch]$SkipCleanup
+)
+
+    . "$PSScriptRoot\PerformanceTestUtilities.ps1"
+
+    If(![string]::IsNullOrEmpty($testDirectoryPath) -And $(GetAbsolutePath $resultsDirectoryPath).StartsWith($(GetAbsolutePath $testDirectoryPath)))
+    {
+        Log "$resultsDirectoryPath cannot be a subdirectory of $testDirectoryPath" "red"
+        exit(1)
+    }
+    
+    if([string]::IsNullOrEmpty($nugetClientFilePath) -Or !$(Test-Path $nugetClientFilePath))
+    {
+        Log "The NuGet client at '$nugetClientFilePath' cannot be resolved. Attempting to use the fallback option." "yellow"
+
+        $relativePath = $([System.IO.Path]::Combine("..", "..", "artifacts", "VS15", "NuGet.exe"))
+        $nugetClientFilePath = GetAbsolutePath $([System.IO.Path]::Combine($PSScriptRoot, $relativePath))
+        if(!$(Test-Path $nugetClientFilePath))
+        {
+            Log "The project has not been built and there is not NuGet.exe available at $nugetClientFilePath. Either build the exe or pass a path to a NuGet client."
+            exit(1)
+        }
+        Log "Using the fallback client from $nugetClientFilePath"
+    }
+
+    if([string]::IsNullOrEmpty($testDirectoryPath))
+    {
+        $testDirectoryPath = $([System.IO.Path]::Combine($env:TEMP,"np"))
```

Comment:
same here, I thought in `%temp%` we used `NuGet`, `NuGetScratch` or `NuGetTestFolder` directories